src/Bookings.js

```diff
@@ -12,7 +12,8 @@ const Bookings = () => {
     <div className="App-content">
       <div className="container">
         <Search search={search} />
-        {/* <SearchResults results={FakeBookings} /> */}
+        <SearchResults results={FakeBookings} />
```

Comment:
this is good that you're passing in your data as props to use in the `SearchResults` component